api/client/samples/anomaly_detection/sample_anomaly_detection.ipynb

```diff
@@ -0,0 +1,784 @@
+{
```

Comment:
<p>period needed after "multiple locations".</p><p> </p><p>Maybe:</p><p>"Since Gro can have multiple sources for a given data series, we will need a function to help us obtain the best data series for multiple locations. We want to choose the best series every time, so we'll use the rank_series_by_source method to retrieve the source with the best data coverage."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>